src/fides/api/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1286,6 +1290,77 @@ def upload_manual_webhook_data(
     )
 
 
+@router.get(
+    PRIVACY_REQUEST_TRANSFER_TO_PARENT,
+    status_code=HTTP_200_OK,
+    dependencies=[Security(verify_oauth_client, scopes=[PRIVACY_REQUEST_TRANSFER])],
+    response_model=Dict[str, Optional[List[Row]]],
+)
+def privacy_request_data_transfer(
+    *,
+    privacy_request_id: str,
+    rule_key: str,
+    db: Session = Depends(deps.get_db),
+    cache: FidesopsRedis = Depends(deps.get_cache),
+) -> Dict[str, Optional[List[Row]]]:
+    """Transfer access request iinformation to the parent server."""
+    privacy_request = PrivacyRequest.get(db=db, object_id=privacy_request_id)
+
+    if not privacy_request:
+        raise HTTPException(
+            status_code=HTTP_404_NOT_FOUND,
+            detail=f"No privacy request with id {privacy_request_id} found",
+        )
+
+    rule = Rule.filter(db=db, conditions=(Rule.key == rule_key)).first()
+    print(rule.__dict__)
```

Comment:
assuming this is for debugging, fine for now but just don't want to forget about it before we merge